### PR TITLE
Portfolio desktop - setup

### PR DESCRIPTION
**Pull request by lRebornsl**
**Portfolio Desktop - Setup**
In this pull I:
- Fixed form inputs as the assignment prompts me.
- Desktop layout already created.

### CODE REVIEW

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @lRebornsl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
- Congratulations 🎉 on the impressive work you've done so far. I've made some suggestions below to help you improve your project so kindly take a look at it and if you don't understand anything please don't hesitate to ask. Best of luck! 🤞🏻.
- I will make my comment right here in the review summary since I don't have access to all the changes in your `styles.css` file. 
- [ ] To create a good user experience while sticking to the `Figma`  design template. Kindly consider adding a `hover` effect on all your buttons.
https://github.com/lRebornsl/portfolio/blob/86bac56e5887c811d8da471504b1f5ec1c81705f/css/styles.css#L296-L298

- [ ] Kindly use the guide below:
Screenshot 2023-05-04 at 15 36 25

**📝NOTE:** Please add the `hover` effect on all your buttons as none of buttons have it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -274,24 +274,12 @@ <h1>About me</h1>
       </div>
       <form action="https://formspree.io/f/xpzebdko" method="POST" class="main-form">
```
Comment (Yerimah) [inline index.html:275]: - [ ] Hey @lRebornsl, 👋 you have done well implementing the project requirements but I would like you to make some adjustments. Only the main title should be wrapped in a `<h1>` tag, other title headings should be wrapped in other `hx` tags for convention, semantics, and SEO reasons.

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

Hi @lRebornsl 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f74385950663341434b656f754c7a454976722f67697068792e676966


 ## Highlights
- No linter error ✅ 
- Descriptive PR title ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @lRebornsl 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Descriptive PR👍
✔ Good commit message👍
✔ Responsive web design👍
### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**